201-logic-app-as2-send-receive/azuredeploy.json

```diff
@@ -0,0 +1,1279 @@
+          "$connections": {
+            "value": {
+              "as2": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]",
+                "connectionId": "[concat(resourceGroup().id, '/providers/Microsoft.Web/connections/', parameters('contoso_AS2_Connection_Name'))]",
+                "connectionName": "[parameters('contoso_AS2_Connection_Name')]"
+              }
+            }
+          }
+        }
+      },
+      "dependsOn": [
+        "[parameters('contosoIntegrationAccountName')]",
+        "[resourceId('Microsoft.Web/connections', parameters('contoso_AS2_Connection_Name'))]"
+      ]
+    },
+    {
+      "name": "[parameters('fabrikamSalesAS2SendLogicAppName')]",
+      "type": "Microsoft.Logic/workflows",
+      "location": "[parameters('logicAppLocation')]",
+      "tags": {
+        "displayName": "Fabrikam Sales AS2 Send"
+      },
+      "apiVersion": "2016-06-01",
+      "properties": {
+        "state": "Enabled",
+        "integrationAccount": {
+          "id": "[resourceId('Microsoft.Logic/integrationAccounts', parameters('fabrikamIntegrationAccountName'))]"
+        },
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+          "actions": {
+            "HTTP": {
+              "inputs": {
+                "method": "POST",
+                "uri": "[listCallbackURL(concat(resourceId('Microsoft.Logic/workflows', parameters('contosoAS2ReceiveLogicAppName')), '/triggers/manual'), '2016-06-01').value]",
+                "headers": {
+                  "AS2-From": "FabrikamSales",
+                  "AS2-To": "Contoso",
+                  "Message-Id": "@guid()",
+                  "content-type": "text/plain"
+                },
+                "body": "Fabrikam Sales - sample message"
+              },
+              "runAfter": { },
+              "type": "Http"
+            }
+          },
+          "parameters": {
+            "$connections": {
+              "defaultValue": { },
+              "type": "Object"
+            }
+          },
+          "triggers": {
+            "Recurrence": {
+              "recurrence": {
+                "frequency": "Hour",
+                "interval": 1
+              },
+              "type": "Recurrence"
+            }
+          },
+          "contentVersion": "1.0.0.0",
+          "outputs": { }
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "as2": {
+                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', parameters('logicAppLocation'), '/managedApis/', 'as2')]",
+                "connectionId": "[concat(resourceGroup().id, '/providers/Microsoft.Web/connections/', parameters('fabrikam_AS2_Connection_Name'))]",
```

Comment:
same here; I would expect there to be some sort of /subscriptions/sub-id/ string at the beginning